### DISCUSSION TITLE
Organizations domains and servermail domains in UI

### DISCUSSION THREAD

**Discussion by mredodos** [postalserver/postal#2666] [Help with using Postal] [answered]
hi , which are the difference between organizations domain and mail server domains in the UI ? (i dont talk about postal.yml) 

![image](https://github.com/postalserver/postal/assets/42175654/156ebb41-4d6f-4e4f-a252-7ef0c1a7a98d)
![image](https://github.com/postalserver/postal/assets/42175654/7af60bf7-6a93-42e9-918b-3423538f271a)

**Comment by willpower232** [comment, marked as answer]
Domains at the Organization level apply to all Servers so if you had a testing or a shared company domain you wanted to try out each Server with then you could set it at the Organization level and not have to worry about all the others.

**Reply by mredodos** [reply]
ok thx
